src/components/Book/BookForm.jsx

```diff
@@ -0,0 +1,23 @@
+import '../../styles/form.css';
+
+const BookForm = () => {
+  const selectors = ['Suzanne Collins', 'Frank Herbert', 'Suzanne Collins'];
+  return (
+    <div className="bookform">
+      <div className="title">ADD NEW FORM</div>
+
+      <div className="form">
+        <input placeholder="Book title" />
+        <select>
+          {
+            selectors.map((selector) => <option key={selector} value={selector}>{selector}</option>)
+        }
+        </select>
```

Comment:
- [ ] You should have 2 inputs, one for the book title and another one for the author because when you will be able to add a book to your list you should enter the title of the book and the author who write the book then you can select the category of the book 😁 , please kindly fix that 🙃